README.md

```diff
@@ -13,3 +13,19 @@ npx install-peerdeps --dev @edx/typescript-config
 ```
 
 This will install the package and all of its peer dependencies.
+
+## Usage
+
+Create file in repository `tsconfig.json`, adding references to the root directory, output directory, and directories to include/exclude in TypeScript compilation.
```

Comment:
Perhaps also include a link to the TypeScript documentation about the tsconfig.json file? https://www.typescriptlang.org/docs/handbook/tsconfig-json.html